packages/examples/pages/org.tsx

```diff
@@ -7,7 +7,7 @@ import { ComponentList } from '../components/ComponentList';
 const Orgs = React.memo(() => {
   const nile = useNile();
   const { isLoading, data: orgs } = useQuery('organizations', () =>
-    nile.listOrganizations()
+    nile.listOrganizations({})
```

Comment:
requiring an empty object seems strange

Reply:
this was a bit hacky to do. I think we'll probably need a full class to generate this in a non-hacky way. 

What I mean by this is the workspace id is required by the API, but not the sdk (since it is handled at initialization). Since handlebars won't do the logic, I think we'd have to add some more flags to get this to work "correctly" so there's a variable for all optional parameters. As it stands now, only `list[Organizations|Entities|Users]` fall under this category, since those "require" the optional workspace, so updated those to have an optional arg.